aws/data_source_aws_dynamodb_table.go

```diff
@@ -15,14 +15,30 @@ func dataSourceAwsDynamoDbTable() *schema.Resource {
 	return &schema.Resource{
 		Read: dataSourceAwsDynamoDbTableRead,
 		Schema: map[string]*schema.Schema{
+			"arn": {
```

Comment:
Reordered the `data_source_aws_dynamodb_table` attributes to match  `resource_aws_dynamodb_table` for easier checking of missing attributes.